src/pudl/output/ferc1.py

```diff
@@ -891,12 +892,16 @@ def drop_other_fuel_types(df):
     fbp_df = (
         fuel_ferc1.pipe(drop_other_fuel_types)
         .pipe(
-            pudl.analysis.classify_plants_ferc1.fuel_by_plant_ferc1,
+            pudl.analysis.record_linkage.classify_plants_ferc1.fuel_by_plant_ferc1,
             fuel_categories=fuel_categories,
             thresh=thresh,
         )
-        .pipe(pudl.analysis.classify_plants_ferc1.revert_filled_in_float_nulls)
-        .pipe(pudl.analysis.classify_plants_ferc1.revert_filled_in_string_nulls)
+        .pipe(
+            pudl.analysis.record_linkage.classify_plants_ferc1.revert_filled_in_float_nulls
+        )
+        .pipe(
+            pudl.analysis.record_linkage.classify_plants_ferc1.revert_filled_in_string_nulls
+        )
         .merge(
```

Comment:
Maybe I'm confused but are these revert function calls still necessary now that the actual ID classifier is run later? All that happens is the fuel columns are merged on so I don't think any nullifying is done yet?

Reply:
In the fuel by plant fuction, there's some `fillna`'s being done that I think do make this necessary. It seems like there's definitely room to refactor the fuel by plant stuff, but I've been hesitant to change anything too much given the large size of this PR already.

Reply:
That makes sense to me about not doing that refactor here! Just wanted to be careful about making "0's" in float columns null when the true value is actually 0.

Reply:
Saw your comment above about using these reverts for legacy reasons.